index.html

```diff
@@ -18,22 +18,125 @@
     </header>
     <main class="main-section">
       <section class="content">
-          <h1>I’m Muhammed Glad to see you!</h1>
-          <p id="primary-text">
-            I’m a software developer! I can help you build a product , feature
-            or website Look through some of my work and experience! If you like
-            what you see and have a project you need coded, don’t hestiate to
-            contactl me.
-          </p>
-          <p class="social-media">LET’S CONNECT</p>
-          <ul class="icons">
-            <li><img src="./assets/Vector.svg" alt="twitter" /></li>
-            <li><img src="./assets/Linkedin icon.svg" alt="linkedin" /></li>
-            <li><img src="./assets/Vector (2).svg" alt="m" /></li>
-            <li><img src="./assets/Vector (1).svg" alt="github" /></li>
-            <li> <img src="./assets/Vector (3).svg" alt="s" /></li>
-          </ul>
-        </div>
+        <h1>I’m Muhammed Glad to see you!</h1>
+        <p id="primary-text">
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contactl
+          me.
+        </p>
+        <p class="social-media">LET’S CONNECT</p>
+        <ul class="icons">
+          <li><img src="./assets/Vector.svg" alt="twitter" /></li>
+          <li><img src="./assets/Linkedin icon.svg" alt="linkedin" /></li>
+          <li><img src="./assets/Vector (2).svg" alt="m" /></li>
+          <li><img src="./assets/Vector (1).svg" alt="github" /></li>
+          <li><img src="./assets/Vector (3).svg" alt="s" /></li>
+        </ul>
+      </section>
+      <section class="portfolio-list-section">
+        <img src="./assets/Snapshoot Portfolio.png" alt="Snapshoot-1" />
+        <p class="project-title">Tonic</p>
+        <ul class="canopy-list">
+          <li id="canopy">CANOPY</li>
+          <li class="title-and-year">Back End Dev</li>
+          <li class="title-and-year">2023</li>
+        </ul>
+        <p class="project-description">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="teck-stacks">
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+        <button class="btn">See Project</button>
+      </section>
```

Comment:
- [ ] While you did a wonderful job styling everything inside your cards :100: :+1: , it is a project requirement to use 'Flexbox' to lay out your elements :+1: . [Here](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) you can find more information about flexbox on CSS.